packages/gestalt-codemods/0.125.0-1.0.0/__testfixtures__/convert-touchable-shape-to-rounding-square.input.js

```diff
@@ -0,0 +1,11 @@
+// @flow
+import React from 'react';
+import { Touchable } from 'gestalt';
+
+export default function Square() {
+  return (
+    <Touchable shape="square">
```

Comment:
@jennyscript Do we also plan on supporting `shape={rounded ? 'rounded' : 'square'}` with the codemod?

The codemod will get a more verbose but you can find an example here: https://github.com/pinterest/gestalt/blob/master/packages/gestalt-codemods/0.111.0-0.112.0/text-remove-semibold.js#L73

An alternative is to update those instances manually

Reply:
Updated to support ternaries